src/openforms/js/components/admin/form_design/registrations/objectsapi/fields/OrganisationRSIN.js

```diff
@@ -0,0 +1,36 @@
+import {useField} from 'formik';
+import {FormattedMessage} from 'react-intl';
+
+import Field from 'components/admin/forms/Field';
+import FormRow from 'components/admin/forms/FormRow';
+import {TextInput} from 'components/admin/forms/Inputs';
+
+const OrganisationRSIN = () => {
+  const [fieldProps] = useField('organisatieRsin');
+  return (
+    <FormRow>
+      {/*TODO: errors*/}
```

Comment:
Will this be addressed in a later PR?

Reply:
No need, it already works :D The validation errors worked much quicker than I anticipated and I forgot about these comments.